main.cpp

```diff
@@ -529,15 +696,20 @@ void Drone::maneuver(float input)
     propellerB.accelerate(input);
     propellerC.accelerate(input);
     propellerD.accelerate(input);
+
+    std::cout << "Manuevering drone!\n";
 }
 
 bool Drone::land()
 {
-    if( gpsA.altitude > 0)
+    if( gpsA.altitude > 0.0f)
     {
         // check the lidar!
+        std::cout << "Landing. Clear landing area!\n";
+        gpsA.altitude -= 1.0f;
         return true;
     }
+    else std::cout << "Can't land. Already on ground!\n";
```

Comment:
If you use curly braces for either the `if` or `else, use them for both
- [ ] revise

Reply:
removed else as well